docs/reference/services.rst

```diff
@@ -4,10 +4,64 @@
 MicroOVN services
 =================
 
+MicroOVN has lots of its functionality separated into distinct services that can
+be controlled via MicroOVN enable/disable or snap/systemd, only the former is
+recommended.
+
 This page presents a list of all MicroOVN services. Their descriptions are
 for reference only - the user is not expected to interact directly with these
 services.
 
+Handling services with enable/disable
+-------------------------------------
+
+The status of all services is displayed by running:
+
+.. code-block:: none
+
+   microovn status
+
+``central service``
+-------------------
+
+This is responsible for the database control. The database is a RAFT database so
+it can handle (n-1)/2 failures, where n is the number of nodes.
+
+Central is enabled on a new node whenever there are less than 3 nodes running
+the central services
+
+This service controls the following snap services:
+
+- ``microovn.ovn-ovsdb-server-nb``
+- ``microovn.ovn-ovsdb-server-sb``
+- ``microovn.ovn-northd``
+
+``chassis service``
+-------------------
+
+This service controls the OVN control and MicroOVN's interaction with OVN.
```

Comment:
Did you mean `ovn-controller` daemon?

What is meant by `MicroOVN's interaction with OVN` ?

For the uninitiated, the use of the word `controller` may be confusing, so it is worth adding a sentence explaining that it is a distributed component running on the side of every Open vSwitch instance. Better examples of wording may be found on the [OVN architecture page](https://manpages.ubuntu.com/manpages/oracular/en/man7/ovn-architecture.7.html)